templates/template-primary-secondary.js

```diff
@@ -0,0 +1,59 @@
+import '../components/colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element';
+
+class TemplatePrimarySecondary extends LitElement {
+
+	static get properties() {
+		return {};
+	}
+
+	static get styles() {
+		return css`
+			.container {
+				display: grid;
+				grid-template-columns: 2fr auto 1fr;
+				grid-template-rows: auto auto auto;
+				grid-template-areas:
+				"header header header"
+				"primary divider secondary"
+				"footer footer footer";
+			}
+			#header {

```

Comment:
We have to be a bit careful with IDs, since for browsers who don't yet support Shadow DOM (coughEdgecough), all of these IDs will be thrown into the global namespace and could collide with others on the page or other CSS on the page.

For that reason, I'd recommend going with CSS classes instead, and namespace the class names with the component name. So they'd become `d2l-template-primary-secondary-header`, `d2l-template-primary-secondary-footer`, and so on. It's a bit of a mouthful, but GZIP to the rescue!